Exiled.API/Extensions/Role.cs

```diff
@@ -5,13 +5,53 @@
 // </copyright>
 // -----------------------------------------------------------------------
 
+using Exiled.API.Enums;
```

Comment:
Your using directives should be inside the namespace.

Reply:
IHateThisButFixed!